src/ConnectionStates/LoginState.cs

```diff
@@ -76,7 +75,7 @@ public override void ProcessInput(string command)
 
         public override string BuildPrompt()
         {
-            return "> ";
+            return string.Format("Please enter your password:{0}> ", Environment.NewLine);
```

Comment:
nit: WheelMUD is inconsistent here so I'll open a separate ticket for going through and fixing this globally in the code, but my rough understanding of the [telnet protocol issues at hand](https://www.freesoft.org/CIE/RFC/1123/31.htm) is:  anything going through Telnet to the user is supposed to send <CR><LF>.  (But anything written to the server console output instead of sent to a user, should probably use Environment.NewLine to ensure proper display on the running platform.)